recipes/npTDMS/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "npTDMS" %}
+{% set version = "0.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd6d131cc386cfc65b02c6ff9653cf311ad17c7d38b2a29d2c513b00394e6cb1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - tdmsinfo = nptdms.tdmsinfo:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - nptdms
+  commands:
+    - tdmsinfo -h
+
+about:
+  home: https://github.com/adamreeve/npTDMS
+  license: LGPL-3.0
+  license_family: GPL
+  # Requested inclusion of license in sdist:
+  # https://github.com/adamreeve/npTDMS/pull/99
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING.LESSER'
```

Comment:
Could you please open an issue in ``npTDMS`` to include the license in the ``MANIFEST.in`` ?